unstructured/file_utils/filetype.py

```diff
@@ -451,7 +451,13 @@ def document_to_element_list(
     for i, page in enumerate(document.pages):
         page_elements: List[Element] = []
         for layout_element in page.elements:
-            element = normalize_layout_element(layout_element)
+            if hasattr(page, "image"):
+                image_format = page.image.format
+                coordinate_system = PixelSpace(width=page.image.width, height=page.image.height)
+            else:
+                image_format = None
+                coordinate_system = None
+            element = normalize_layout_element(layout_element, coordinate_system=coordinate_system)
```

Comment:
@qued I moved assignment of the coordinate system to the beginning of the loop in order to pass the coordinate system to `normalize_layout_element`. Otherwise, `normalize_layout_element` calls element constructors without the coordinate system in context; the element's constructor can't instantiate `CoordinatesMetadata` without both the points and system present because we want to avoid creating `CoordinatesMetadata` with points and system from different contexts that may not be in sync.